routes/_fake/admin.tsx

```diff
@@ -0,0 +1,157 @@
+                window.location.href = "/_fake/admin"
+              })
+              .catch((error) => {
+                console.error("Error creating thing:", error)
+                alert(`Error creating thing: ${error.message}`)
+              })
+          }}
+        >
+          <div className="mb-2">
+            <label className="block text-gray-700 text-sm font-bold mb-1">
+              Name:
+              <input type="text" name="name" className="w-full" required />
+            </label>
+          </div>
+          <div className="mb-2">
+            <label className="block text-gray-700 text-sm font-bold mb-1">
+              Description:
+              <input
+                type="text"
+                name="description"
+                className="w-full"
+                required
+              />
+            </label>
+          </div>
+          <button
+            type="submit"
+            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+          >
+            Create Thing
+          </button>
+        </form>
+      </div>
+
+      {/* Display thing resources */}
+      <div className="mb-8">
+        <h2 className="text-xl font-bold mb-4">Thing Resources</h2>
+        <Table
+          rows={things.map((thing) => ({
+            ...thing,
+            actions: (
+              <form
+                action="/things/delete"
+                method="POST"
+                onSubmit={(e) => {
+                  e.preventDefault()
+                  if (
+                    confirm(`Are you sure you want to delete "${thing.name}"?`)
+                  ) {
+                    fetch("/things/delete", {
+                      method: "POST",
+                      headers: { "Content-Type": "application/json" },
+                      body: JSON.stringify({ thing_id: thing.thing_id }),
+                    })
+                      .then((response) => {
+                        if (!response.ok)
+                          throw new Error(
+                            `HTTP error! status: ${response.status}`,
+                          )
+                        return response.json()
+                      })
+                      .then(() => {
+                        window.location.href = "/_fake/admin"
+                      })
+                      .catch((error) => {
+                        alert(`Error deleting thing: ${error.message}`)
+                      })
+                  }
+                }}
+              >
+                <input type="hidden" name="thing_id" value={thing.thing_id} />
+                <button
+                  type="submit"
+                  className="inline-flex items-center px-2.5 py-1.5 border border-transparent text-xs font-medium rounded text-white bg-red-600 hover:bg-red-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-500"
+                >
+                  Delete
+                </button>
+              </form>
```

Comment:
this looks really bad because of the double outline, remove the outline surrounding the button by overriding the form style, you can add a custom form class if needed